Cargo.toml

```diff
@@ -33,13 +33,18 @@ tokio = { version = "0.2", features = ["rt-core", "sync"] }
 trust-dns-resolver = "0.18.1"
 async-trait = "0.1.21"
 unicode-width = "0.1.8"
+netstat2 = "0.9.0"
+sysinfo = "0.15.1"
 
-[target.'cfg(target_os="linux")'.dependencies]
-procfs = "0.7.4"
 
 [dev-dependencies]
 insta = "0.12.0"
 pnet_base = "0.26.0"
 cargo-insta = "0.11.0"
 packet-builder = "0.5.0"
 regex = "1"
+
+[build-dependencies]
+#[target.'cfg(target_os="windows")'.build-dependencies]
```

Comment:
Is it correct, those crates should not be downloaded on unixes.

Reply:
@imsnif actually that's not the case: I had to uncomment this line to make sure that `http_req` doesn't get downloaded.

Reply:
Oi, I'm sorry about that! You are very right and I managed to miss this. :) I'm fixing this now and will push a patch. Thanks bringing this up (and catching it so quickly)!